python/example_code/lambda/hello/hello_lambda.py

```diff
@@ -0,0 +1,29 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# snippet-start:[python.example.hello_lambda]
+
+import boto3
+
+
+def main():
+    """
+    List the Lambda functions in your AWS account.
+    """
+    # Create the Lambda client
+    lambda_client = boto3.client("lambda")
+
+    # Use the paginator to list the functions
+    paginator = lambda_client.get_paginator("list_functions")
+    response_iterator = paginator.paginate()
+
+    print("Here are the Lambda functions in your account:")
+    for page in response_iterator:
+        for function in page["Functions"]:
+            print(f"  {function['FunctionName']}")
```

Comment:
Should we limit this to something like 5? They could have a lot of Lambda functions.

Reply:
IMO, I find it confusing to arbitrarily limit the number of functions displayed to 5. If they have a hundred Lambda functions, they should probably see them! It also shows the paginator in action.